bin/iotaJsonTester.js

```diff
@@ -135,33 +136,36 @@ function exitClient() {
 }
 
 var commands = {
-    'config': {
+    config: {
         parameters: ['host', 'port', 'apiKey', 'deviceId'],
-        description: '\tConfigure the client to emulate the selected device, connecting to the given host.',
+        description:
+            '\tConfigure the client to emulate the selected device, connecting to the given host.',
```

Comment:
I have a kind of deja vu looking to this... :)

We were discussing about line length in https://github.com/telefonicaid/perseo-fe/pull/314#issuecomment-459038580 and we ended with `"printWidth": 120` (https://github.com/telefonicaid/perseo-fe/blob/master/.prettierrc.json#L6). I think this setting should be common to this an other similar opened PRs.

Reply:
(I didn't realize in the IOTA Sigfox onboarding PR... the 100 at https://github.com/telefonicaid/sigfox-iotagent/blob/master/.prettierrc.json#L6 probably should be 120 but it's fine given that the PR there is already merged).

Reply:
Fixed f90c273